_docs/render.py

```diff
@@ -0,0 +1,202 @@
+from __future__ import annotations
+
+import inspect
+import json
+import re
+import sys
+from contextlib import contextmanager
+from functools import lru_cache
+from inspect import getsource
+from pathlib import Path
+from types import FunctionType
+from typing import Dict, Optional, Set
+
+import yaml
+from jinja2 import Environment, PackageLoader, select_autoescape
+from magicgui._magicgui import MagicFactory
+
+from npe2 import PluginManager
+from npe2.manifest import PluginManifest
+from npe2.manifest.contributions import ContributionPoints
+from npe2.manifest.utils import Executable
+
+DOCS = Path(__file__).parent
+TEMPLATES = DOCS / "templates"
+_BUILD = DOCS / "_build"
```

Comment:
does this need to get updated? 

```suggestion
_BUILD = DOCS.parent / "docs" / "plugins"
```

Reply:
sure, that would prevent the need to always run it with a command line path like `python _docs/render.py docs/plugins`